htdocs/contrat/card.php

```diff
@@ -1134,6 +1136,7 @@
 			// Replicate extrafields
 			$objectsrc->fetch_optionals($originid);
 			$object->array_options = $objectsrc->array_options;
+            if(!empty($objectsrc->ref_client)) $ref_customer = $objectsrc->ref_client;
```

Comment:
ref_customer is used to fill the reference of contract defined by customer.
Here you put the reference of an order or proposal into the contract.
The ref of the order or proposal must stay on the order and proposal and contract can be linked to this proposal or contract but the field that reference a contract must not contains a value that is related to something else.